automation/run-tests.sh

```diff
@@ -60,17 +71,34 @@ function install_nmstate {
 }
 
 function run_tests {
-    docker_exec "
-      cd /workspace/nmstate &&
-      pytest \
-        --verbose --verbose \
-        --log-level=DEBUG \
-        --durations=5 \
-        --cov=$PYTHON27_PATH/libnmstate \
-        --cov=$PYTHON27_PATH/nmstatectl \
-        --cov-report=html:htmlcov-py27 \
-        tests/integration \
-    ${nmstate_pytest_extra_args}"
+    if [ $TEST_TYPE == "all" ] || [ $TEST_TYPE == "tox_code_style" ];then
```

Comment:
I believe this needs double quotes around `$TEST_TYPE` in case it would be be defined (even though this should not happen it wold avoid thinking about this when reading this).